TrafficCapture/dockerSolution/build.gradle

```diff
@@ -85,6 +90,12 @@ trafficComparatorServices.forEach {projectName, dockerImageName ->
         from 'python:3.10.10'
         runCommand("apt-get update && apt-get install -y netcat lsof")
         copyFile("setup.py", "/setup.py")
+        copyFile("setup.py", "containerTC/setup.py")
+        copyFile("cli.py", "containerTC/cli.py")
+        copyFile("traffic_comparator", "containerTC/traffic_comparator")
+        if (projectName == 'jupyterNotebook') {
+            copyFile("ResultsRepository.ipynb", "containerTC/ResultsRepository.ipynb")
+        }
```

Comment:
I'm interested in how this is going to play out w/ the volume mount...
But independent of that - why not copy with a wildcard or directory?  Having to include every file here is going to be a painful tax.